localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -0,0 +1,38 @@
+﻿l_english:
+ epidemic_conversion_bonus:0 ""Promote Culture" [councillor_task|E] Speed: #P +50%#!"
+
+ ##############
+ # Tribal Era #
+ ##############
+
+ innovation_amphibious:0 "Amphibious"
+ innovation_amphibious_desc:0 "We surface on land only when we please, but being natives to the water we can travel the seas at ease"
```

Comment:
The innovation description ends with a dot.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>